modules/gatsby/package.json

```diff
@@ -72,16 +73,21 @@
     "prismjs": "^1.15.0",
     "probe.gl": "^3.0.1",
     "prop-types": "^15.6.2",
+    "react": "^16.8.6",
```

Comment:
These were previously moved to `peerDependencies` to let the application control React versions.

Reply:
reverted to that